src/com/epam/javatrainings/annotations/validation/DtoValidationService.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.javatrainings.annotations.validation;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DtoValidationService {
+    private DtoValidationService() {
+        throw new IllegalStateException("No instances");
+    }
+
```

Comment:
can define a map which will provide us needed annotationProcessor
private static Map<Class<?>, ? extends AnnotationProcessor> annotationProcessorMap = Map.of(
            Length.class, new LengthAnnotationProcessor(),
            Adulthood.class, new AdulthoodAnnotationProcessor(18),
            Email.class, new EmailAnnotationProcessor(),
            Min.class, new MinAnnotationProcessor(),
            Max.class, new MaxAnnotationProcessor()
    );